bot/beltbot.py

```diff
@@ -66,38 +67,36 @@
 
 
 @BOT.command(name="request", help=_request_help)
-async def request_handler(ctx, colour, *body):
+async def request_handler(ctx, *, body):
     if ctx.message.channel.name != "belt-requests":
         await ctx.send("Only available in #belt-requests.")
         return
 
-    colour = "".join(char for char in colour if char not in PUNCTUATION)
-
-    if colour not in ALL_BELTS:
-        await ctx.send(
-            (
-                f"{ctx.message.author.mention} I don't think {colour} is a real belt :( \n"
-                f"Try one of these: {HUMAN_READABLE_BELTS}"
-            ),
-            mention_author=True,
-        )
-        return
-
-    role_name = ALL_BELTS[colour]["name"]
-
+    belt_regex = "^(?P<belt>{belts})(?P<reason>.*)".format(belts="|".join(ALL_BELTS))
```

Comment:
Move this somewhere else. Maybe the constants.py?

We are recreating this regex every call. Only needs to be done once on application start.